tests/test_setup.py

```diff
@@ -0,0 +1,58 @@
+import os
+from pathlib import Path
+import re
+import tempfile
+import unittest
+
+
+class TestSetup(unittest.TestCase):
```

Comment:
I guess the real goal of this test is to "make sure that the PyPI distribution works".

I would argue that this is tested more strongly (and with less lines of code / maintenance needs) by adding a travis test that creates the package distribution, installs it (e.g. like [this](https://stackoverflow.com/a/36014474/1069467)) and then runs either the whole test suite or some subset of it.

Reply:
True. We could definitely have a GH Actions workflow that does this.